File: app/src/main/java/com/chess/cryptobot/model/response/bittrex/BittrexPrice.java
package com.chess.cryptobot.model.response.bittrex;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

class BittrexPrice {
    @SerializedName("Quantity")
    @Expose
    private Double quantity;
    @SerializedName("Rate")
    @Expose
    private Double rate;

    Double getQuantity() {
        return quantity;
    }

    Double getRate() {
        return rate;
    }
}

File: app/src/main/java/com/chess/cryptobot/model/response/bittrex/BittrexPrice.kt
package com.chess.cryptobot.model.response.bittrex

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class BittrexPrice {
    @SerializedName("quantity")
    @Expose
    val quantity: Double = 0.0
    @SerializedName("rate")
    @Expose
    val rate: Double = 0.0

}